roles/deploy/tasks/deploy.yml

```diff
@@ -72,6 +72,20 @@
     - skip_ansible_lint
   notify: restart puma
 
+# Ensure the correct Node version is activated
+
+- name: register ruby version in new build
+  slurp:
+    src: "{{ build_path }}/.node-version"
+  register: deploy_node_version
+  become: yes
+
+- name: install node version (and yarn)
+  include_role:
+    name: nodenv_install
+  vars:
+    node_version: "{{ deploy_node_version['content'] | b64decode | trim }}"
+
```

Comment:
This is clever. But why don't we use the new script (other PR) for this?

Reply:
Main reason: I got carried away when I finally got into the groove with Ansible, and forgot about the script! This does have the advantage of reporting the version number to the console.
Also I figured it would be safer to re-use the method already used here for rbenv, which never actually chdir's into the build path. I think it probably would have been fine to chdir and run the script though, and it would have been better to favour shell scripting over ansible more..